cmd/token.go

```diff
@@ -9,10 +9,12 @@ import (
 	"github.com/spf13/viper"
 )
 
-var isUserToken bool
-var userScopes string
-var revokeToken string
-var overrideClientId string
+var (
+    isUserToken      bool
+    userScopes       string
+    revokeToken      string
+    overrideClientId string
+)
```

Comment:
The author of the repository has his own style of coding and does not need to change it